todo_app/app.py

```diff
@@ -1,26 +1,99 @@
+import os
 from flask import Flask, redirect, url_for,  render_template
 from todo_app.data import session_items
 from todo_app.flask_config import Config
 from flask import request
+import requests
 
 app = Flask(__name__)
-app.config.from_object(Config)
+app.config.from_object(Config)   
+
+KEY=os.getenv("KEY")
+TOKEN=os.getenv("TOKEN")
+    
+
+class TransportAPI:
+    def __init__(self):
+        api_keys = get_application_keys()
+        self.payload = api_keys
+
 
 @app.route('/index')
 def index():
-    list = session_items.get_items()
+    list = getAllToDoFromTrell0()
     return  render_template('index.html', list=list)
 
-@app.route('/hello/<name>')
-def hello(name):
-   return 'welcome %s' % name
+def getAllToDoFromTrell0():
+   id = '60af9248e87283184d346aa8'
+   list= []
+   trelloList= requests.get(f'https://api.trello.com/1/boards/{id}/cards', params={'key': KEY, 'token': TOKEN}).json()
+   for card in trelloList:
+      if card ["idList"] =="60af9248e87283184d346aa9":
+         status = "ToDo"
+      elif card ["idList"]=="60af9248e87283184d346aaa":
+         status = 'Doing'   
+      else:
+         status = "Done"
+      list.append({"status":status, "id":card["id"], "title":card["name"]})
+   return list  
+
+def createBoard():
```

Comment:
I don't think this function is ever used anywhere - it might turn out useful in the next module so I'm happy for you to leave it but in practice we should remove any old code like this - we can easily recover it from git if needed

Reply:
Removed